spec/db_cleaner_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'fileutils'
+
+RSpec.describe Geordi::DBCleaner do
+  describe '#new' do
+    let(:dir) { File.join(Dir.pwd, 'spec') }
```

Comment:
Please write to ./tmp in tests. Maybe tmp is already excluded in the gitignore file?

Reply:
Good point, and it is indeed already excluded :+1: